src/borg/testsuite/archiver/debug_cmds.py

```diff
@@ -63,6 +63,55 @@ def test_debug_put_get_delete_obj(self):
         output = self.cmd(f"--repo={self.repository_location}", "debug", "delete-obj", "invalid")
         assert "is invalid" in output
 
+    def test_debug_id_hash_format_put_parse_obj(self):
+        self.cmd(f"--repo={self.repository_location}", "rcreate", RK_ENCRYPTION)
+        data = b"some data"
+        meta = b'{"some" : "property"}'
+        meta_dict = json.loads(meta)
+
+        self.create_regular_file("file", contents=data)
+        self.create_regular_file("meta.json", contents=meta)
```

Comment:
you could use data.bin and meta.json to make more clear what's in these files.

Reply:
could be even plain.bin and (below) object.bin to make the difference between plain (uncompressed, not encrypted, ...) data and a "formatted" repo object more clear in the filenames.